docs/DeveloperGuide.md

```diff
@@ -331,6 +331,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2a1. FlashBack shows an error message
       
       Use case resumes at step 1
+    
+**Use case: Find cards by search criteria**
+
+**MSS**
+
+1. User requests to find cards by criteria with given keywords
+1. FlashBack shows a list of flashcards matching given keywords according to search criteria
```

Comment:
Should this be step 2 instead of step 1?

Reply:
It is okay, the documentation will be reflected as step 1 and step 2 correctly.